builtin/providers/vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -740,7 +770,7 @@ func resourceVSphereVirtualMachineDelete(d *schema.ResourceData, meta interface{
 	return nil
 }
 
-func waitForNetworkingActive(client *govmomi.Client, datacenter, name string) resource.StateRefreshFunc {
+func waitForNetworkingActive(client *govmomi.Client, datacenter, name string, waitForNetworkInterfaces bool) resource.StateRefreshFunc {
```

Comment:
That wasn't merged, when I started my work and we need this feature to start using terraform. But I think it could replace this change and additionally all `WaitForIP` and `waitForNetworkingActive` calls.